onnxruntime/test/platform/ios/ios_package_test/README.md

```diff
@@ -0,0 +1,28 @@
+# iOS End-to-End Test App for ORT-Mobile
+
+This End-to-End test app for iOS will test ORT Mobile C/C++ API framework using CoocaPods
+
+## Requirements
+
+- [CocoaPods](https://cocoapods.org/)
+
+## Build and Test iOS Framework using build.py
+
+Use the [build for iOS simulator](http://www.onnxruntime.ai/docs/how-to/build/android-ios.html#cross-build-for-ios-simulator) with `--build_apple_framework`
+
+## Run the iOS End-to-End Test App standalone
+
+### Requirements
+
+- A pre-built ORT Mobile iOS framework
+
+### Steps
+
+1. Update the [OnnxRuntimeBase.podspec](./OnnxRuntimeBase.podspec), replace `${ORT_BASE_FRAMEWORK_FILE}` with the path of the pre-built ORT Mobile iOS framework
+2. Go to this folder
```

Comment:
Where do I get the pre-built ORT Mobile iOS framework? Doesn't seem to be covered by this doc. Can we link to some other doc or add details here?

Reply:
The doco update for gh-pages is not yet merged, update this one for now